test/system/fidelity_test_all.py

```diff
@@ -0,0 +1,18 @@
+import subprocess
+import sys
+
+# Check if the correct number of arguments are provided
+if len(sys.argv) != 3:
+    print("Usage: python script.py <input_file> <directory>")
+    sys.exit(1)
+
+# Store the path parameters
+original_file = sys.argv[1]
+directory = sys.argv[2]
+
+# Run the scripts
+subprocess.run(["python3", "fidelity_test_01.py", directory])
```

Comment:
this assumes "fidelity_test_0x.py" is in the current working directory. if this script is launched not in `test/system`, then it will fail.

Reply:
I’ve updated the script to check if the fidelity_test_0x.py files exist before running them. If they’re not in the same folder as the script, you can now provide an optional third argument to specify the directory where the test files are located. This should make it easier to run the tests from different locations without running into missing file errors.